### PR TITLE
Added License file

### PR DESCRIPTION
**Pull request by Ayush9026**
## Description 

This PR solves issue https://github.com/theupdateframework/community/issues/7

### CODE REVIEW

**Comment by lukpueh** [comment]
Why MIT? The issue explicitly mentions Apache 2.0. Also, maybe we can use CSL for this repo like we do for the TUF spec? (see https://github.com/theupdateframework/specification/pull/289)

cc @JustinCappos

**Comment by Ayush9026** [comment]
Okay @lukpueh sir i will add which @JustinCappos sir tell.

**Comment by JustinCappos** [comment]
TUF's reference implementation is dual licensed, with both MIT and Apache 2.0.  However, this isn't a source code repository.  So it really doesn't make sense to think about a source code license here.  The community license is also not the specification.   (I'd appreciate your help ensuring that the spec repo is correctly CSL 1.0 licensed and follows all of the documented steps for having contributions). 

I think that since this repo only contains documentation, it should be CC 4.0 International BY licensed or similar.  Please propose this on the #tuf channel and see if anyone objects.

**Review by JustinCappos — CHANGES_REQUESTED** [review]
Get consensus that CC 4.0 International BY is correct (or similar)

**Comment by Ayush9026** [comment]
@JustinCappos sir i asked in #tuf channel [here] (https://cloud-native.slack.com/archives/C8NMD3QJ3/p1716554431827259) did not get any response.

**Comment by JustinCappos** [comment]
Okay, please switch to CC-BY-4.0

**Comment by Ayush9026** [comment]
Thank you @JustinCappos sir for the clarification. I have updated the license to Creative Commons Attribution 4.0 International (CC-BY-4.0) as requested. Please review the changes, and let me know if everything aligns with your requirements

**Review by JustinCappos — APPROVED** [review]

**Comment by JustinCappos** [comment]
One thing to check, do we need a SPDX license header or similar for this?

**Comment by Ayush9026** [comment]
Thank you @JustinCappos sir for raising this concern. I will await your confirmation regarding the SPDX license header before proceeding with the addition. Once you've reviewed and approved, I'll ensure it's included in the License file accordingly.

**Comment by JustinCappos** [comment]
Okay, please look for best practices, etc. and see what we should do here.
I'd be happy to learn the right way from what you find.  I'm not sure if
the SPDX header is only applied to individual files, to the license itself,
or something else.

**Comment by h4l0gen** [comment]
Hi @JustinCappos , I researched about this, First I looked for what CNCF wants from us. So they clearly stated [here ](https://github.com/cncf/techdocs/blob/main/docs/repo-setup.md#license-files)that 
`Unless otherwise specified, documentation for CNCF projects is licensed under CC-BY-4.0. Code is licensed under Apache 2.0.`
But I also gathered information from these CNCF graduated projects:

- [Knative](https://github.com/knative/community/blob/main/LICENSE-docs)
- [Kubernetes](https://github.com/kubernetes/community/blob/master/LICENSE)
- [Cilium](https://github.com/cilium/community/blob/main/LICENSE)
- [Argo ](https://github.com/orgs/argo/repositories?type=all)
- [Cloudevents](https://github.com/orgs/cloudevents/repositories?type=all)(all repo contains Apache 2.0)
- [Containerd](https://github.com/containerd/?q=&type=all&language=&sort=)
- [etcd](https://github.com/etcd-io/website/blob/main/LICENSE)
- [Falco ](https://github.com/falcosecurity/community/blob/main/LICENSE)

Only [Knative ](https://github.com/knative/community/blob/main/LICENSE-docs)and [etcd ](https://github.com/etcd-io/website/blob/main/LICENSE)contains CC-BY-4.0 license on their community/docs/website repository, else uses [Apache-2.0 ](https://www.apache.org/licenses/LICENSE-2.0)same as CNCF-techdocs.

Also from [knative's CC-BY-4.0 license file](https://github.com/knative/community/blob/main/LICENSE-docs) and this [etcd.io license ](https://github.com/etcd-io/website/blob/main/LICENSE)created by @nate-double-u (honourable member of CNCF/techdocs team itself) we can be assure that `CC-BY-4.0` license does not require SPDX header. 

So my vote is also to use `CC-BY-4.0` license without SPDX header, as per [CNCF guidelines.](https://github.com/cncf/techdocs/blob/main/docs/repo-setup.md#license-files)

SPDX header can be use in Apache-2.0 license, [only Knative uses](https://github.com/knative/community/blob/main/LICENSE) that, they had whole discussion on this [here](https://github.com/knative/community/issues/970). Other graduated project from above list uses Apache-2.0 license same as cncf/techdocs (without SPDX header)

`Side-note`:  SPDX header is only applied to license itself. @JustinCappos 

I hope my effort will help us to move forward. Thank you 👍

**Comment by h4l0gen** [comment]
Above research made me to think, that [theupdateframework.io](https://github.com/theupdateframework/theupdateframework.io) also needs `Apache 2.0` and `CC-BY-4.0` license as written [here](https://github.com/cncf/techdocs/blob/main/docs/repo-setup.md#license-files) that, 
```
Most CNCF documentation repositories are a mix of code (website code) and documentation itself, so they need two license files.
```
So I am raising issue [here ](https://github.com/theupdateframework/theupdateframework.io/issues/60) for website license and If @JustinCappos and others agree then I will create PR in [python-tuf](https://github.com/theupdateframework/python-tuf/issues) too, to add another `CC-BY-4.0` license.

File: LICENSE-MIT.txt [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+     example, because of any applicable exception or limitation to
+     copyright--then that use is not regulated by the license. Our
+     licenses grant only permissions under copyright and certain
+     other rights that a licensor has authority to grant. Use of
+     the licensed material may still be restricted for other
+     reasons, including because others have copyright or other
+     rights in the material. A licensor may make special requests,
+     such as asking that all changes be marked or described.
+     Although not required by our licenses, you are encouraged to
+     respect those requests where reasonable. More_considerations
```
Comment (h4l0gen) [inline LICENSE-MIT.txt:51]: Hi @Ayush9026, a small typo in `More_consideration`, line should be  
`respect those requests where reasonable. More considerations`
Thank you 👍

**Review by Vivekgaddigi — APPROVED** [review]

File: LICENSE-MIT.txt [hunk 2]
```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```
Comment (h4l0gen) [inline LICENSE-MIT.txt:1]: We are not using MIT LICENSE, hence we should file name from `LICENSE-MIT.txt`  to LICENSE.txt or similar.
@Ayush9026 WYT?

**Pull request closed without merging**